internal/manifests/pod/build_test.go

```diff
@@ -1,15 +1,22 @@
-package kibana
+package pod
 
 import (
 	"testing"
 
 	"github.com/openshift/elasticsearch-operator/internal/utils"
+
 	corev1 "k8s.io/api/core/v1"
 )
 
-// Call this method if you want to test that podSpec's node selectors contain only the linux one.
-// Note: This method gets called from other tests. That is why we made it public to other packages.
-func CheckIfThereIsOnlyTheLinuxSelector(podSpec corev1.PodSpec, t *testing.T) {
```

Comment:
why was this removed?

Reply:
This is not really removed but inlined into the one single test function using it. The diff is awkwardly broken here. Please check the `pod/build_test.go` in isolation.